CodingChallenges/CC_046_Asteroids/Processing/CC_046_Asteroids/Laser.pde

```diff
@@ -0,0 +1,48 @@
+// Daniel Shiffman
+// http://codingtra.in
+// http://patreon.com/codingtrain
+// Code for: https://youtu.be/hacZU523FyM
+// Processing transcription: Chuck England
+
+class Laser {
+  PVector pos;
+  PVector vel;
+
+  Laser(PVector spos_, float angle_) {
+    pos = new PVector(spos_.x, spos_.y);
+    vel = PVector.fromAngle(angle_);
+    vel.mult(10);
+  }
+
+  void update() {
+    pos.add(vel);
+  }
+  
+  void render() {
+    pushMatrix();
+    stroke(255);
+    strokeWeight(4);
+    point(pos.x, pos.y);
+    popMatrix();
+  }
+
+  Boolean hits(Asteroid asteroid) {
```

Comment:
`boolean`?

Reply:
Ack! Sometimes my keyboard does strange things I did not intend. :) I will fix this.